config/config.go

```diff
@@ -39,7 +39,7 @@ var (
 
 	defaultMaxPayloadSize = ByteSize(1 << 50)
 
-	defaultRetryNumber = 0
+	defaultRetryNumber = 1
```

Comment:
why is the default value to 1? if you do that the behavior of chproxy will change after the update? If yes, it would be better to put it back to 0 then let the chproxy admin choose whether or not the feature should be active

Reply:
I agree with @mga-chka. For the sake of test it's enough what you have done in config_test.go